docs/flamegraphs.md

```diff
@@ -0,0 +1,9 @@
+# Flamegraphs
+
+To help in identifying performance bottlenecks and slow performing functions, Developer Tools includes a CPU time [flamegraph](http://www.brendangregg.com/flamegraphs.html) of the current page's execution. Open the Altis Developer Tools and navigate to the "Xray -> Flamegraph" panel.
+
+![](./qm-flamegraph.png)
+
+The flamegraph shows execution time along the X axis. The vertical axis displays the call-depth of the PHP stack at that point in time. The flamegraph can tell you very quickly what is using the most time in the current page generation. Items in the flamegraph as clickable to zoom-in on the specific call.
```

Comment:
```suggestion
The flamegraph shows execution time along the X axis. The vertical axis displays the call-depth of the PHP stack at that point in time. The flamegraph can tell you very quickly what is using the most time in the current page generation. Items in the flamegraph are clickable to zoom-in on the specific call.
```

Reply:
One thing that wasn't super clear to me was clicking at the bottom of the trace to zoom back out again. Would be good to note that here too.